### DISCUSSION TITLE
Example request: uwebsocket.js integration with Graphql

### DISCUSSION THREAD

**Discussion by byteab** [uNetworking/uWebSockets.js#507] [Show and tell]
is there any example on how to integrate uwebsocket.js with Graphql. 
I wanna Integrate it with Graphql-jit to maximize performance. would you mind to please provide a sample repo.

**Comment by dalisoft** [comment]
@EhsanSarshar See [here](https://github.com/nanoexpress/middlewares/tree/master/packages/graphql) or [here](https://github.com/sifrr/sifrr/tree/master/packages/server/sifrr-server#graphql-server)

**Reply by byteab** [reply]
Good job and thank you
